libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/utils/RouteUtils.kt

```diff
@@ -219,20 +219,15 @@ class RouteUtils {
     fun findCurrentBannerInstructions(
         currentStep: LegStep?,
         stepDistanceRemaining: Double
-    ): BannerInstructions? {
-        currentStep?.bannerInstructions()?.let {
+    ) = currentStep?.bannerInstructions()?.let {
             val instructions: List<BannerInstructions> = sortBannerInstructions(it)
-            for (instruction in instructions) {
-                val distanceAlongGeometry = instruction.distanceAlongGeometry().toInt()
-                if (distanceAlongGeometry >= stepDistanceRemaining.toInt()) {
-                    return instruction
-                }
+            instructions.firstOrNull { instruction ->
+                instruction.distanceAlongGeometry().toInt() >= stepDistanceRemaining.toInt()
```

Comment:
looks weird, not sure that it's equal to previous approach 🤔

Reply:
Identical to previous logic